manifests/router/link_route_pattern.pp

```diff
@@ -10,7 +10,7 @@
 #
 define qpid::router::link_route_pattern(
   $prefix    = 'queue.',
-  $connector = 'broker',
+  $connector = '',
```

Comment:
Better to use UNSET or undef instead of empty string? I am just basing this off of other places that do things like this.